src/main/resources/avro/bdg.avdl

```diff
@@ -1165,6 +1165,12 @@ record Feature {
    */
   union { null, string } featureId = null;
 
+
+  /**
+   Identifier for this sample.
+   */
+  union { null, string } sampleId = null;
```

Comment:
@heuermh do you think this should be an Avro Sample or just a string sampleId?

Reply:
Elsewhere we use `sampleId`, with the expectation that users will join on sampleId against  metadata `samples` &rarr; `Sample.sampleId`.

Reply:
Yup.  I'd suggest updating the doc string to something like
```java
  /**
   Sample identifier for this feature. Join with Sample.sampleId for sample metadata.
   sampleId tag in GFF3.
  */
```
This matches the style used in #108.